docs/intl.md

```diff
@@ -0,0 +1,155 @@
+# Spark Regional Date Time Support
+
+## Spark Components support regional date functionality
+
+- Spark components have been updated to support regional date functionality in latest versions of **V3** and **V4**.
+- The new updates in Spark are internally feature toggled so that we can release the new date/time regional at the same time for all apps
+- There are two methods for enabling the new regional date features in Spark components: a global variable, and a feature toggle in the form of a cookie
+  o To test the new Spark component regional date formatting in your application, set the global variable on the window object:
+
+      ```
+      window['GUX_OPTIONS_enableRegionalDates']= true
+      ```
+
+## Feature Toggling
+
+- All regional date functionality should be feature toggled. We want to avoid confusing customers with different date time behavior across applications
+- The Spark component functionality is internally feature toggled and when it is time to GA this feature, will be globally enabled via a cookie set in web directory
+- If your app resides outside of web directory, this cookie should still take effect after the user navigates to web directory the first time
+- All regional date functionality that is outside of the Spark components should be feature toggled behind the feature toggle for the epic, `PURE-2946_InternationalizationOfDatesAndTimes`
+
+## Using components for new date/time regional features
+
+All Spark components that relate to date or time leverage support the new regional date/time features. Here is a list of all the components that are affected by these changes:
```

Comment:
Maybe this: `All Spark components that relate to date or time will leverage support of the new regional date/time features.`